### ISSUE TITLE
Various statement dump conversion issues

### ISSUE THREAD

**Issue by mmd-osm** [drolbr/Overpass-API#188]
## Loss of CSV formatting options

Example: http://overpass-turbo.eu/s/7a3  

Convert to Overpass QL/compact QL,etc  via Export -> ... -> convert to Overpass QL.

Result: 

``` C
[bbox:44.77793589631623,4.06494140625,55.32914440840507,24.9169921875]
[out:csv]              // <<<< parameters got lost
[timeout:3600]
;
node["amenity"="pharmacy"]->.pharmacies;
foreach.pharmacies(
  out body;
  (coord-query:);
  area._["boundary"="postal_code"];
  out body;
);
```

As we don't have an Overpass XML representation for out:csv parameters, this may require some further rework. Solution depends quite a bit on #136.

**Comment by jotpe** [comment]
Example?

**Comment by jotpe** [comment]
I see.

**Comment by mmd-osm** [comment]
Also, `is_in` is not handled properly, `[~"key"~"value"]` drops the key, role name in `rel(bw:"street")` gets dropped.

**Comment by mmd-osm** [comment]
We have some more candidates with similar issues:

##  Key Regex
* `way[highway="path"][~"name"~"yellow"];`
* `way[highway="path"][~"name"~"yellow",i];`

Both are converted to `way["highway"="path"][~"yellow"];`

**Comment by drolbr** [comment]
Finally fixed in b1f8bd50161500d4ea83be0741a84a935968d5b4
